hw/ip/entropy_src/data/entropy_src_testplan.hjson

```diff
@@ -9,9 +9,83 @@
   entries: [
     {
       name: sanity
-      desc: '''Enable entropy_src. Verify entropy, status, interrupt bits.'''
+      desc: '''Enable entropy_src in LFSR mode, wait for interrupt, verify entropy.'''
       milestone: V1
       tests: ["entropy_src_sanity"]
     }
+    {
+      name: efuse
+      desc: '''Verify entropy is not available via register if efuse input is 0.'''
+      milestone: V2
+      tests: [""]
```

Comment:
nit: leave the `tests` key as an empty list `[]` as opposed to a list with single empty string entry.